retail/interactive-tutorial/RetailEvents.Samples/ImportUserEventsBigQuerySample.cs

```diff
@@ -0,0 +1,118 @@
+﻿// Copyright 2022 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START retail_import_user_events_from_big_query]
+// Import user events into a catalog from BigQuery using Retail API
+
+using Google.Cloud.Retail.V2;
+using Google.LongRunning;
+using System;
+
+/// <summary>
+/// The import user events BigQuery sample class.
+/// </summary>
+public class ImportUserEventsBigQuerySample
+{
+    private const string DataSetId = "user_events";
+    private const string DataSchema = "user_event";
+    private const string TableId = "events";
```

Comment:
This shouldn't be harcoded inside the region tag. You can harcode them in the method that executes the sample down below, but for documentation, they should be parameters (and apply the same reasoning to the other samples even if I missed it).

Reply:
done